elytron/src/test/resources/org/wildfly/extension/elytron/domain-test.xml

```diff
@@ -89,4 +89,12 @@
             <permission class-name="org.wildfly.security.auth.permission.LoginPermission"/>
         </permission-set>
     </permission-sets>
+    <http>
+        <http-authentication-factory name="management-http-authentication-MyDomain" security-domain="MyDomain" http-server-mechanism-factory="global" />
```

Comment:
@Mergers: These configurations are necessary to wire up the Security Domain services used in this test case. I'm not sure it is the correct approach here.